Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -1513,27 +1513,35 @@ public static partial class PythonOps {
         /// <summary>
         /// Python runtime helper for raising assertions. Used by AssertStatement.
         /// </summary>
-        /// <param name="msg">Object representing the assertion message</param>
-        public static void RaiseAssertionError(CodeContext context, object msg) {
+        public static void RaiseAssertionError(CodeContext context) {
             PythonDictionary builtins = context.GetBuiltinsDict() ?? context.LanguageContext.BuiltinModuleDict;
 
-            object obj;
-            var message = String.Empty;
-
-            if (msg != null) {
-                message = PythonOps.ToString(msg);
+            if (builtins._storage.TryGetValue("AssertionError", out object obj)) {
+                if (obj is PythonType type) {
+                    throw PythonOps.CreateThrowable(type);
+                }
+                throw PythonOps.CreateThrowable(DynamicHelpers.GetPythonType(obj));
             }
 
-            if (builtins._storage.TryGetValue("AssertionError", out obj)) {
-                PythonType type = obj as PythonType;
-                if (type != null) {
-                    throw PythonOps.CreateThrowable(type, message);
-                } else {
-                    throw PythonOps.CreateThrowable(DynamicHelpers.GetPythonType(obj), message);
+            throw PythonOps.AssertionError("{0}");
```

Comment:
Just for my benefit, can you add a comment on why we call this with a format specifier?

Reply:
Because I just mindlessly removed the message attribute. There shouldn't be an argument there... Will fix tomorrow.